risc0/zkvm/src/fault_ids.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright 2023 RISC Zero, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Warning: this code is generated by cargo xtask and should not be modified by
+// hand.
+
+/// The image ID of the fault checker guest program
+#[rustfmt::skip]
+pub const FAULT_CHECKER_ID: [u32; 8] = [73690331, 82127537, 5319644, 3512993428, 3332701599, 1824104521, 2226309199, 2308040824];
```

Comment:
Was this produced with the reproducible build system? Probably in part 2, but we should have some mechanism of CI to ensure the fault checker ELF and ID are built reproducibly and that they do not get stale.